pages/index.html

```diff
@@ -62,20 +62,28 @@ <h2 class="block">Corporate Members</h2>
 
 		<h2 class="block">Other jQuery Foundation Projects</h2>
 		<section class="project-tiles row">
-			<div class="project-tile six columns color secondary-orange">
-				<a href="//jqueryui.com" class="jqueryui small logo">jQueryUI</a>
-			</div>
-			<div class="project-tile six columns color secondary-green">
-				<a href="//jquerymobile.com" class="jquery-mobile small logo">jQuery Mobile</a>
-			</div>
+			<a href="//jqueryui.com">
+				<div class="project-tile six columns color secondary-orange">
+					<div class="jqueryui small logo"></div>
+				</div>
+			</a>
+			<a href="//jquerymobile.com">
+				<div class="project-tile six columns color secondary-green">
+					<div class="jquery-mobile small logo"></div>
+				</div>
+			</a>
 		</section>
 		<section class="project-tiles row">
-			<div class="project-tile six columns color qunit-secondary-purple">
-				<a href="//qunitjs.com" class="qunitjs small logo">QUnit</a>
-			</div>
-			<div class="project-tile six columns color sizzle-red">
-				<a href="//sizzlejs.com" class="sizzlejs small logo">Sizzle</a>
-			</div>
+			<a href="//qunitjs.com">
+				<div class="project-tile six columns color qunit-secondary-purple">
+					<div class="qunitjs small logo"></div>
+				</div>
+			</a>
+			<a href="//sizzlejs.com">
+				<div class="project-tile six columns color sizzle-red">
+					<div class="sizzlejs small logo"></div>
+				</div>
+			</a>
 		</section>
 	</section>
```

Comment:
@rxaviers Don't link to HTML 4 specs. :) They may be seriously outdated, a lot of things changed in HTML5 (including this one AFAIK).